lib/bot.js

```diff
@@ -131,6 +140,30 @@ Bot.prototype.parseText = function(text) {
     });
 };
 
+Bot.prototype.parseMessage = function(message) {
+  return message
+
+    // bold
+    .replace(/(\u0002)([^\u0002]+)(\u0002|\u000f|$)/g, function(match, opener, text, end) {
+      return '*' + text + '*';
+    })
+
+    // italic
+    .replace(/(\u001d)([^\u001d]+)(\u001d|\u000f|$)/g, function(match, opener, text, end) {
+      return '_' + text + '_';
+    })
+
+    // underlined
+    .replace(/(\u001f)([^\u001f]+)(\u001f|\u000f|$)/g, function(match, opener, text, end) {
+      return '_' + text + '_';
```

Comment:
Is underline to italics the best way to go? Maybe it'd be better to just let it stay unformatted, not sure.